package.json

```diff
@@ -41,7 +41,7 @@
     "jest": "^23.6.0",
     "less": "^2.7.3",
     "node-sass": "^4.13.1",
-    "rollup": "^1.1.2",
+    "rollup": "^1.19.0",
```

Comment:
we have updated `rollup` to the latest version

https://github.com/egoist/rollup-plugin-postcss/blob/1e1022d9555850ac87584c73e741d171059946b4/package.json#L44

Reply:
Now it's on `^2.34.2`. @alexdilley would you be able to rebase and drop this file from the PR?